tm2/pkg/bft/node/node.go

```diff
@@ -843,11 +844,9 @@ func makeNodeInfo(
 	genDoc *types.GenesisDoc,
 	state sm.State,
 ) (p2p.NodeInfo, error) {
-	txIndexerStatus := "on"
-	if _, ok := txIndexer.(*null.TxIndex); ok {
-		txIndexerStatus = "off"
-	} else if txIndexer == nil {
-		txIndexerStatus = "none"
+	txIndexerStatus := "off"
+	if txIndexer.GetType() != null.IndexerType {
+		txIndexerStatus = "on"
 	}
```

Comment:
nit: should we use constants for the available indexerStatus?

Reply:
Yes, I ran into this exact problem on the PR - some values were expected, but not made clear. Added 💯 

[8f13f4e](https://github.com/gnolang/gno/pull/546/commits/8f13f4e39e4882ea9a7e4490349edb0c58d29a4b)